app/views/imports/_form_main.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="overflow-auto text-nowrap">
```

Comment:
so this differs slightly from the design, but i tried to balance the tasks of the ticket with what we needed for parity with Publish. so i used a table to display imports, and moved the importer over to the `form_status` partial, to keep things all on one view.

Reply:
I think this view makes a lot of sense.  But - I also think we're going to need `podcast_imports#show`, to track the breakdown of episode statuses, show errors, etc.

Doesn't need to be anything real fance... just a turbo-frame with some version of this, and the polling-controller to refresh it until complete.

<img src="https://github.com/PRX/feeder.prx.org/assets/1410587/fd8bde76-495c-43f2-a3a6-f295bd3014d6" width=500/>

This is also where you show "duplicate guids" in the RSS feed and any errors (here's [the publish code](https://github.com/PRX/publish.prx.org/blob/main/src/app/series-import-status-card/series-import-status-card.component.html#L34) fwiw)

Reply:
Alternatively, you could add these to the `#index` table you've got.  But it might look kind of crowded?  Feel free to try it out though.  The things I'm looking for are:

- The entries in RSS feed
- Successfully imported count
- In progress count
- Remaining count
- Errors count
- Duplicate guids count
- And some sort of visual-spinner/progressbar indicating things are still happening